demo/package.json

```diff
@@ -4,16 +4,18 @@
   "private": true,
   "dependencies": {
     "@xbyorange/mercury": "1.0.0",
-    "@xbyorange/mercury-api": "1.0.1",
+    "@xbyorange/mercury-api": "1.1.0",
     "@xbyorange/mercury-browser-storage": "1.0.0",
     "@xbyorange/react-mercury": "file:../",
+    "@xbyorange/mercury-memory": "1.0.0",
     "lodash": "^4.17.11",
     "prop-types": "^15.7.2",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.3",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "react-router": "^4.3.1"
```

Comment:
There is no need to install both `react-router` and `react-router-dom`. You can use the `Route` component from `react-router-dom` directly.